snapshots/output/syntax/src/inheritance.ts

```diff
@@ -136,11 +132,11 @@
     consumesInterface({
 //  ^^^^^^^^^^^^^^^^^ reference syntax 1.0.0 src/`inheritance.ts`/consumesInterface().
       interfaceMethod: (): string => 'inferred',
-//    ^^^^^^^^^^^^^^^ definition syntax 1.0.0 src/`inheritance.ts`/interfaceMethod1:
+//    ^^^^^^^^^^^^^^^ definition syntax 1.0.0 src/`inheritance.ts`/interfaceMethod0:
```

Comment:
This case is still unhandled. It's a simpler than `returnStatementInsideArgumentExpression` so it could be a good one to look at first

Reply:
I'll look at this first then :)